binding.gyp

```diff
@@ -9,10 +9,11 @@
         "<!@(node -p \"require('node-addon-api').include\")"
       ],
       "cflags!": [ "-fno-exceptions", "-fno-rtti" ],
-      "cflags_cc!": [ "-fno-exceptions", "-fno-rtti" ],
-      "cflags_cc": [ "-std=c++17", "-fexceptions", "-frtti" ],
+      "cflags_cc!": [ "-std=gnu++17", "-fno-exceptions", "-fno-rtti" ],
+      "cflags_cc": [
+        "<!@(pkg-config --cflags parquet arrow)", "-Wall", "-g", "-std=c++17", "-fexceptions", "-frtti" ],
```

Comment:
Is it absolutely required to add everything before `std=c++17` here?

Reply:
double-checked, and it must have been made obsolete by something else I tried. [d365710](https://github.com/Comparative-Inc/comparative-parquet/pull/8/commits/d36571040677307b263a746350bae2eee2fc236b)